x-pack/plugins/apm/server/routes/fleet.ts

```diff
@@ -170,15 +172,34 @@ const createCloudApmPackagePolicyRoute = createApmServerRoute({
     if (!hasRequiredRole || !cloudApmMigrationEnabled) {
       throw Boom.forbidden(CLOUD_SUPERUSER_REQUIRED_MESSAGE);
     }
-    return {
-      cloud_apm_package_policy: await createCloudApmPackgePolicy({
-        cloudPluginSetup,
-        fleetPluginStart,
-        savedObjectsClient,
-        esClient,
-        logger,
-      }),
-    };
+
+    const cloudApmAackagePolicy = await createCloudApmPackgePolicy({
+      cloudPluginSetup,
+      fleetPluginStart,
+      savedObjectsClient,
+      esClient,
+      logger,
+    });
+
+    const [setup, internalSavedObjectsClient] = await Promise.all([
+      setupRequest(resources),
+      core
+        .start()
+        .then(({ savedObjects }) => savedObjects.createInternalRepository()),
+    ]);
+
+    const spaceId = plugins.spaces?.setup.spacesService.getSpaceId(request);
+
+    // force update the index pattern title with data streams
+    await createStaticIndexPattern(
+      setup,
+      config,
+      internalSavedObjectsClient,
+      spaceId,
+      true
+    );
```

Comment:
Booleans as arguments are hard to read. As a reader, I'm wondering what's being toggled on here.

Instead:

```suggestion
    await createStaticIndexPattern({
      setup,
      config,
      internalSavedObjectsClient,
      spaceId,
      overwrite: true
    });
```